ansible_events_ui/schemas.py

```diff
@@ -57,9 +57,19 @@ class Extravars(BaseModel):
 class Activation(BaseModel):
     id: Optional[int]
     name: StrictStr
+    description: Optional[StrictStr]
+    execution_env_id: int
     rule_set_file_id: int
     inventory_id: int
+    restart_policy_id: int
+    playbook_id: int
+    activation_status: Optional[StrictStr]
+    activation_enabled: bool
     extra_var_id: int
+    last_restarted: Optional[datetime]
+    restarted_count: Optional[int]
+    created: Optional[datetime]
```

Comment:
Since the `created` and `last_modified` timestamps are generated automatically, they should not be included in model used for validating client request at all.

Reply:
I added these here coz I thought we might need them when calling GET request to get the activation object and pass it on to UI. What do you think?

Reply:
Yes, we do need a response model as well, but it should be a separate model for a response:
https://fastapi.tiangolo.com/tutorial/response-model/#add-an-output-model

Response model should include `id` field along with `created` and `last_modified` timestamps.